exasol/analytics/query_handler/python_query_handler_runner.py

```diff
@@ -71,9 +71,7 @@ def run(self) -> ResultType:
             try:
                 self._handle_finish()
             except Exception as e1:
-                LOGGER.exception(
-                    "Catched exeception during cleanup after an exception."
-                )
+                LOGGER.exception("Caught exeception during cleanup after an exception.")
```

Comment:
```suggestion
                LOGGER.exception("Caught exception during cleanup after an exception.")
```

Reply:
OMG - 2 typos in one error message! 
Thanks!
Good (well) catch!